README.md

```diff
@@ -10,7 +10,13 @@ To build the tool, simply clone the repo, and run `build.cmd` in command line or
 - `JS.fs`: handles the emitting of the `domWeb.js` and `domWindows.js`
 
 ## Contribution Guide
-The `dom.generated.d.ts` and `webworker.generated.d.ts` files that are used in TypeScript repo `master` branch are used as baselines. For each pull request, we will try run the script and compare the generated files with the baseline files. So to make the tests passed, please update the baseline and the code at the same pull request. 
+The `dom.generated.d.ts` and `webworker.generated.d.ts` files that are used in TypeScript repo `master` branch are used as baselines. 
```

Comment:
used in **the** TypeScript repository

I think you can remove "`master` branch" - it's mostly implied